frontend/ui/font.lua

```diff
@@ -2,11 +2,10 @@
 Font module.
 ]]
 
-local Device = require("device")
 local Freetype = require("ffi/freetype")
 local lfs = require("libs/libkoreader-lfs")
 local logger = require("logger")
-local Screen = Device.screen
+local CanvasContext = require("document/canvascontext")
```

Comment:
Given that this font.lua module is only used by UI widgets, and not by document rendering wrapping code, shouldn't we keep using `Screen`?

Reply:
That's what I thought as well, but there is a surprising dependency between this module and document code. I am planning to dig more into this and see if we can couple that.